src/gui/src/clockWidget.h

```diff
@@ -466,7 +466,11 @@ class ClockWidget : public QDockWidget, sta::dbNetworkObserver
   void setLogger(utl::Logger* logger);
   void setSTA(sta::dbSta* sta);
 
-  void saveImage(const std::string& clock_name, const std::string& path);
+  void saveImage(const std::string& clock_name,
+                 const std::string& path,
+                 const std::string& corner,
+                 int width_px,
+                 int height_px);
```

Comment:
std::optional?

Reply:
sure, I assume we don't want this at the gui.h too because the swig wrapper wouldn't be right for python? If I recall correctly what SWIG doesn't really know what to do with optional

Reply:
Yeah swig doesn't like optional